code/network/multi.cpp

```diff
@@ -1299,11 +1299,11 @@ void multi_do_frame()
 		int idx;
 		for(idx=0;idx<MAX_PLAYERS;idx++){
 			if(MULTI_CONNECTED(Net_players[idx]) && (Net_player != &Net_players[idx])){
-				if((Multi_client_update_times[idx] < 0) || timestamp_elapsed_safe(Multi_client_update_times[idx], 1000)){
+				if ( !Multi_client_update_times[idx].isValid() || ui_timestamp_elapsed_safe(Multi_client_update_times[idx], 1000) ) {
```

Comment:
Might want to use `MILLISECONDS_PER_SECOND` here.

Reply:
Fixed